### DISCUSSION TITLE
DOMAIN-ERROR

### DISCUSSION THREAD

**Discussion by suryabhankh24** [ONDC-Official/developer-docs#37] [Q&A]
Hi @92shreyansh,

I am getting below response,
{
    "message": {
        "ack": {
            "status": "NACK"
        }
    },
    "error": {
        "type": "DOMAIN-ERROR",
        "code": "130",
        "path": null,
        "message": "https://ftcash.com : Domain verification file (ondc-site-verification.html) is not found "
    }
}
I have already hosted ondc-site-verification.html, is there anything i am missing?

**Comment by 92shreyansh** [comment]
Please use the specific thread for staging on-boarding Q&A https://github.com/ONDC-Official/developer-docs/discussions/31
